packages/flex-plugin-scripts/src/utils/package.ts

```diff
@@ -0,0 +1,38 @@
+import { resolve } from './require';
+import { join } from 'path';
+
+export interface PackageDetail {
+  name: string;
+  found: boolean;
+  package: {
+    name?: string;
+    version?: string;
+  };
+}
+
+/**
+ * @param packages
+ */
+/* istanbul ignore next */
+export const getPackageDetails = (packages: string[]): PackageDetail[] => {
```

Comment:
Can we actually move this into the flex-dev-utils then?

Reply:
spoke offline: to be done in v4